production/loki-mixin/dashboards/loki-chunks.libsonnet

```diff
@@ -54,16 +53,17 @@ local utils = import 'mixin-utils/utils.libsonnet';
         },
       ],
     } +
-    g.dashboard('Loki / Chunks')
+    $.dashboard('Loki / Chunks')
```

Comment:
I'd suggest staying away from the global `$` if you can; it can cause some _fun_ issues with jsonnet and also makes it hard to determine which function it's using.

What was the reason behind this change? AFAICS consistency with other dashboards?

Reply:
So that we can use `addClusterSelectorTemplates(false)` which is defined in the mixin's `dashboard-utils.libsonnet`, which gives us the link dropdown: https://github.com/grafana/loki/blob/main/production/loki-mixin/dashboards/dashboard-utils.libsonnet#L32-L43

I haven't written any jsonnet in ~1y so if you know a better way/possible refactor I'm all ears!

Reply:
Ah, ok. Have a look at line 1:
`local utils = import 'mixin-utils/utils.libsonnet';`

You can import a file to a local var, and then use that.

However it seems like we're peppering a bunch of files with `$` so let me not hold you up on this.